package.json

```diff
@@ -149,6 +149,7 @@
     "rxjs": "7.5.6",
     "tslib": "2.3.0",
     "webdriver-manager": "12.1.7",
+    "yarn": "1.22.19",
```

Comment:
probably you installed this package locally instead of globally, so now we have changes in `yarn.lock` also but we shouldn't

Reply:
reverted